code/modules/mob/mob.dm

```diff
@@ -474,14 +474,25 @@
 
 /mob/living/blind_examine_check(atom/examined_thing)
 	//need to be next to something and awake
-	if(!in_range(examined_thing, src) || incapacitated())
+	if(!Adjacent(examined_thing) || incapacitated())
 		to_chat(src, "<span class='warning'>Something is there, but you can't see it!</span>")
 		return FALSE
-	//also neeed an empty hand, and you can only initiate as many examines as you have hands
-	if(LAZYLEN(do_afters) >= usable_hands || get_active_held_item())
+
+	//you can examine things you're holding directly, but you can't examine other things if your hands are full
+	/// the item in our active hand
+	var/active_item = get_active_held_item()
+	if(active_item && active_item != examined_thing)
+		to_chat(src, "<span class='warning'>Your hands are too full to examine this!</span>")
+		return FALSE
+
+	//you can only initiate exaimines if you have a hand, it's not disabled, and only as many examines as you have hands
+	/// our active hand, to check if it's disabled/detatched
+	var/obj/item/bodypart/active_hand = has_active_hand()? get_active_hand() : null
+	if(!active_hand || (active_hand?.is_disabled()) || LAZYLEN(do_afters) >= usable_hands)
```

Comment:
```suggestion
	if(!active_hand || active_hand.bodypart_disabled || LAZYLEN(do_afters) >= usable_hands)
```
No need to check if `active_hand` exists, since we just checked it.
I'm refactoring `is_disabled()` to nuke it from existence, use `bodypart_disabled` instead.